crates/sui/src/keytool.rs

```diff
@@ -804,6 +821,58 @@ impl KeyToolCommand {
                 .await?;
                 CommandOutput::ZkLoginSignAndExecuteTx(ZkLoginSignAndExecuteTx { tx_digest })
             }
+
+            KeyToolCommand::ZkLoginSigVerify {
+                sig,
+                tx_bytes,
+                provider,
+                curr_epoch,
+            } => {
+                match GenericSignature::from_bytes(
+                    &Base64::decode(&sig).map_err(|e| anyhow!("Invalid base64 sig: {:?}", e))?,
+                )? {
+                    GenericSignature::ZkLoginAuthenticator(zk) => {
+                        if tx_bytes.is_none() || provider.is_none() {
+                            return Ok(CommandOutput::ZkLoginSigVerify(format!(
+                                "Parsed zkLogin signature: {:?}",
+                                zk
+                            )));
+                        }
+                        println!("Parsed zkLogin signature: {:?}", zk);
```

Comment:
Do you need to print this information for this user, or is it more of a tracing/debug kind of thing? If it's needed, then I'd suggest to put it as a field into the `CommandOutput::ZkLoginSigVerify` type, and output a table with all the needed information.